assets/js/components/KeyMetrics/MetricsSelectionPanel/Notice.js

```diff
@@ -70,6 +73,33 @@ export default function Notice() {
 		}
 	} );
 
+	const previousHasMissingCustomDimensions = usePrevious(
+		!! hasMissingCustomDimensions
```

Comment:
I just noticed this doesn't need the `!!` coercion, as its `useSelect()` will only return a boolean. I'll update it myself to save the round trip.

```suggestion
		hasMissingCustomDimensions
```